neural_mfg.py

```diff
@@ -165,6 +163,48 @@ def sample_room(self):
 
         return pd.DataFrame(X_b.numpy()), pd.DataFrame(X_in.numpy()), pd.DataFrame(X_out.numpy())
     
+    def re_sample_room(self):
```

Comment:
A cosa serve questo metodo ? Non c'è modo di usare il metodo sample_room già definito in precedenza ? per evitare di appesantire il codice

Reply:
La differenza è il numero di punti. Nel resampling abbiamo un numero N = 10000 di punti "candidati" ad essere inclusi nel dataset. Nel sample_room usiamo invece gli attributi self.N_b e self.N_in. C'è sicuramente un modo più elegante per scriverlo come unico metodo, te lo cambio